pkg/storage/unified/sql/sqltemplate/mocks/SQLTemplate.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.43.1. DO NOT EDIT.
+// Code generated by mockery v2.53.4. DO NOT EDIT.
```

Comment:
this is the file generated when you run mockery -- alternatively, we could update the arguments to make a non Iface version, but I think this is better.